nrcatalogtools/waveform.py

```diff
@@ -1,9 +1,10 @@
-import lal
-import numpy as np
 import os
+
 import h5py
+import lal
+import numpy as np
 from sxs import WaveformModes as sxs_WaveformModes
-
+from nrcatalogtools.lvc import GetNRToLALRotationAngles
```

Comment:
Change to

```
from nrcatalogtools.lvc import get_nr_to_lal_rotation_angles
```

Reply:
In my opinion, relative imports can bring in inconsistencies, especially for us.

Reply:
Agree with @vaishakp. Lets avoid relative imports as much as possible.

Reply:
OK, I am fine with that. @vaishakp however could you please elaborate what inconsistencies you are talking about?